sdk/src/main/java/co/omise/android/api/Client.kt

```diff
@@ -2,28 +2,44 @@ package co.omise.android.api
 
 import android.os.Build
 import android.os.Handler
-import co.omise.android.BuildConfig
+import co.omise.android.api.exceptions.ClientException
 import co.omise.android.models.Model
-import okhttp3.*
+import okhttp3.CertificatePinner
+import okhttp3.ConnectionSpec
+import okhttp3.OkHttpClient
+import okhttp3.TlsVersion
 import java.util.concurrent.Executor
 import java.util.concurrent.Executors
 import java.util.concurrent.TimeUnit
 
+
 /**
  * Client is the main entry point to the SDK and it needs to be supplied with a public Key. You can use the Client to send a [Request].
  *
  * @param publicKey The key with the `pkey_` prefix.
  * @see Request
  */
-class Client(private val publicKey: String) {
+class Client(publicKey: String?)
```

Comment:
Why is the publicKey a Nullable? I think publicKey is a required value

Reply:
Oh yeah, leftout `?` from when I was using both, should be removed.